src/pages/ContractsPage.js

```diff
@@ -1,58 +1,116 @@
+import {
+  RIGHT_POLICYHOLDERCONTRACT_SEARCH,
+  RIGHT_POLICYHOLDERCONTRACT_CREATE,
+  RIGHT_POLICYHOLDERCONTRACT_UPDATE,
+  RIGHT_POLICYHOLDERCONTRACT_APPROVE,
+} from "../constants";
 import ContractSearcher from "../components/ContractSearcher";
 import { Fab } from "@material-ui/core";
 import AddIcon from "@material-ui/icons/Add";
 
-const styles = theme => ({
-    page: theme.page,
-    fab: theme.fab
-})
+const styles = (theme) => ({
+  page: theme.page,
+  fab: theme.fab,
+});
 
 class ContractsPage extends Component {
-    onAdd = () => historyPush(this.props.modulesManager, this.props.history, "contract.route.contract");
+  onAdd = () =>
+    historyPush(
+      this.props.modulesManager,
+      this.props.history,
+      "contract.route.contract"
+    );
 
-    contractUpdatePageUrl = contract => `${this.props.modulesManager.getRef("contract.route.contract")}${"/" + decodeId(contract.id)}`;
+  contractUpdatePageUrl = (contract) =>
+    `${this.props.modulesManager.getRef("contract.route.contract")}${
+      "/" + decodeId(contract.id)
+    }`;
 
-    onDoubleClick = (contract, newTab = false) => {
-        const { rights, modulesManager, history } = this.props;
-        if (rights.includes(RIGHT_POLICYHOLDERCONTRACT_UPDATE) || rights.includes(RIGHT_POLICYHOLDERCONTRACT_APPROVE)) {
-            historyPush(modulesManager, history, "contract.route.contract", [decodeId(contract.id)], newTab);
-        }
-    }
-    
-    render() {
-        const { intl, classes, rights } = this.props;
-        return (
-            rights.includes(RIGHT_POLICYHOLDERCONTRACT_SEARCH) && (
-                <div className={classes.page}>
-                    <Helmet title={formatMessage(this.props.intl, "contract", "contracts.page.title")} />
-                    <ContractSearcher
-                        onDoubleClick={this.onDoubleClick}
-                        contractUpdatePageUrl={this.contractUpdatePageUrl}
-                        rights={rights}
-                    />
-                    {rights.includes(RIGHT_POLICYHOLDERCONTRACT_CREATE) && withTooltip(
-                        <div className={classes.fab} >
-                            <Fab color="primary" onClick={this.onAdd}>
-                                <AddIcon />
-                            </Fab>
-                        </div>,
-                        formatMessage(intl, "contract", "createButton.tooltip")
-                    )}
-                </div>
-            )
-        )
+  onDoubleClick = (contract, newTab = false) => {
+    const { rights, modulesManager, history } = this.props;
+    if (
+      rights.includes(RIGHT_POLICYHOLDERCONTRACT_UPDATE) ||
+      rights.includes(RIGHT_POLICYHOLDERCONTRACT_APPROVE)
+    ) {
+      historyPush(
+        modulesManager,
+        history,
+        "contract.route.contract",
+        [decodeId(contract.id)],
+        newTab
+      );
     }
+  };
+
+  componentDidMount = () => {
+    const moduleName = "contact";
```

Comment:
I think @olewandowski1 here you can add to the consts this variable because there is 'const' file with all constants 
you can add this one as:
    `export const MODULE_NAME = "contact";`
and import this const into this file